scripts/build_coverage.sh

```diff
@@ -93,7 +93,7 @@ if [ "$skip_gn" == false ]; then
 fi
 
 mkdir -p "$COVERAGE_ROOT"
-lcov --initial --capture --directory "$OUTPUT_ROOT/obj" --output-file "$COVERAGE_ROOT/lcov_base.info"
-lcov --capture --directory "$OUTPUT_ROOT"/obj --output-file "$COVERAGE_ROOT/lcov_test.info"
+lcov --initial --capture --directory "$OUTPUT_ROOT/obj/src" --exclude="$PWD"/third_party/* --exclude=/usr/include/* --output-file "$COVERAGE_ROOT/lcov_base.info"
```

Comment:
Could you add these changes to https://github.com/project-chip/connectedhomeip/blob/master/scripts/build/builders/host.py#L361 as well?

Reply:
Instead of "PWD" is it more correct to do "CHIP_ROOT"? or is this supposed to be run from other directories than chip root?

Reply:
"CHIP_ROOT"  does not works since it is relative path and we need to specify the absolute path here